docs/Instrumentation.md

```diff
@@ -6,48 +6,61 @@ Instrumentation is prototyped in onnx-mlir and can be used to debug runtime issu
+    =nnpaAfterOnnxToZhigh                           -   [NNPA] Profile for onnx and zhigh ops.
+    =nnpaAfterZhighToZlow                           -   [NNPA] Profile for zlow ops.
+
+  --instrument-ops=<string>                         - Specify regex for ops to be instrumented:
+                                                      "NONE" or "" for no instrument,
+                                                      "regex1,regex2, ..." for the specified ops.
+                                                      e.g. "onnx.,zhigh." for onnx and zhigh ops.
+                                                      e.g. "onnx.Conv" for onnx Conv ops.
+
+  Specify what instrumentation actions at runtime:
+      --InstrumentBeforeOp                             - insert instrument before op,
+      --InstrumentAfterOp                              - insert instrument after op,
+      --InstrumentReportTime                           - instrument runtime reports time usage,
+      --InstrumentReportMemory                         - instrument runtime reports memory usage.
 ```
-For example, `Debug/bin/onnx-mlir --instrument-onnx-ops="Conv" --InstrumentBeforeOp --InstrumentAfterOp --InstrumentReportTime mymodel.onnx`
- will instrument before and after each onnx.Conv op and report time usage  when the model is executed. 
 
 Currently, the call of initialization, OMInstrumentInit, need to be added before you load the dynamic library. It is being considered to add it to the beginning of main_graph by compiler. 
 
 ## Run with instrumentation
 Run the model in the same way as usual.
 The instrumentation library will print out the time and memory usage along at each instrumentation point.
-For example, a model, `mymodel.onnx`, is compiled with `Debug/bin/onnx-mlir  --instrument-onnx-ops="ALL" --InstrumentAfterOp --InstrumentReportMemory --InstrumentReportTime mymodel.onnx`.
+For example, a model, `mymodel.onnx`, is compiled with `onnx-mlir  --instrument-stage=afterOnnxToOnnx --instrument-ops="onnx." --InstrumentAfterOp --InstrumentReportMemory --InstrumentReportTime mymodel.onnx`.
 Its runtime output is listed below:
 
 ```
-#  0) after  op= Transpo Time elapsed: 0.000766 accumulated: 0.000766 VMem: 156608 (model/transpose1)
-#  1) after  op= Constan Time elapsed: 0.005398 accumulated: 0.006164 VMem: 156608
-#  2) after  op= Constan Time elapsed: 0.004225 accumulated: 0.010389 VMem: 156608
-#  3) after  op=    Conv Time elapsed: 0.360213 accumulated: 0.370602 VMem: 156608 (model/conv1)
-#  4) after  op= Softplu Time elapsed: 0.190591 accumulated: 0.561193 VMem: 156608 (model/softplus1)
-#  5) after  op=    Tanh Time elapsed: 0.115314 accumulated: 0.676507 VMem: 156608 (model/tanh1)
-#  6) after  op=     Mul Time elapsed: 0.022779 accumulated: 0.699286 VMem: 156608 (model/mul1)
+#  0) after onnx.Transpose Time elapsed: 0.000766 accumulated: 0.000766 VMem: 156608 (model/transpose1)
```

Comment:
Reduce `Time elapsed` to `Elapsed` and give more space for op name. Does it make sense to print one digit (from 6 to 7) for the time? I forgot what's the precision of the clock tick.

Reply:
> Does it make sense to print one digit (from 6 to 7) for the time? 

It seems `gettimeofday()` gives microseconds. So, current 6 digits may be appropriate.